### ISSUE TITLE
customEnv

### ISSUE THREAD

**Issue by meaku** [peerigon/dynamic-config#8]
@Vispercept can you describe why you want a customEnv? 

I'm thinking of handling it the same way as `defaultEnv` so we'd have something like `envArgumentName` which is `env` by default and can be changed to whatever you like. 

It should be applied on reading from `argv` and `env` if you change it so the behavior is consistent.

**Comment by Vispercept** [comment]
Ja servus Michi!

Our intention was to load a config according to a special env-variable.

```
.
|-- config/
|    |-- team1
|    |-- team2
|    |-- team3
|    |-- ......
```

So now you would export an env-variable like `export team="team1"`. I don't want to export something like **env** or **node_env** because we run our program as a commandline-tool and we don't want to generate interferences with other programs.

Your Idea sounds very nice! Would be great if you'd implement it. For our purposes the current state is sufficient.

**Comment by Vispercept** [comment]
Would yo mind to publish the changes to npm by the way?

**Comment by meaku** [comment]
I changed your PR a little bit, so it's `envName` now, which defaults to "env". 

Simple set `dynamicConfig.options.envName = "team1"`. 

It's released as 0.2.0. Let me know if it works for you :) 
Thanks for using `dynamic-config`. 

Pfiadi Thomas!

**Comment by Vispercept** [comment]
That perfectly works for us :-) Thanks a lot!

Habe die Ehre ;-)
